packages/common/src/domain/currency_type_ui_labels.ts

```diff
@@ -0,0 +1,6 @@
+export enum CurrencyTypeUILabels {
+    ETHEREUM = "Ethereum",
+    BITCOIN = "Bitcoin",
+}
+
+export const currencyTypeUILabels = [CurrencyTypeUILabels.BITCOIN, CurrencyTypeUILabels.ETHEREUM] as const;
```

Comment:
Nit: Screaming snake case for constants

```suggestion
export const CURRENCY_TYPE_UI_LABELS = [CurrencyTypeUILabels.BITCOIN, CurrencyTypeUILabels.ETHEREUM] as const;
```

Reply:
We might have the same syntax in other similar files. Definitely something we should look at later this sprint since it'll be a small refactor.